web/server.js

```diff
@@ -39,31 +38,24 @@ module.exports = function(options) {
 
   var server = new Hapi.Server(serverConfig);
 
-  server.connection({
-    host: options.host,
-    port: options.port
-  });
-
   if ( options.logging ) {
-    server.register({
-      register: require('hapi-bunyan'),
+    await server.register({
+      plugin: require('hapi-bunyan'),
       options: {
         logger: require('bunyan').createLogger({
           name: 'id-webmaker-org',
           level: options.logLevel
         })
       }
-    }, function(err) {
```

Comment:
we don't try catch any of these anywhere since the previous logic basically just threw the error which is what will happen now as well.